code/datums/tips/tips.dm

```diff
@@ -107,7 +104,7 @@ GLOBAL_LIST_EMPTY(tips_by_category)
 		"Ian is the best department pet.",
 		"Crusher is the best department pet.",
 		"Columbo is the best department pet.",
-		"Don't lie to yourself, when was the last time a Research director took Ives out?",
```

Comment:
And this?

Reply:
This one wasn't removed, just altered to actually make sense. The joke set up I did there was each department pet being "the best department pet", except for Ives who no one takes out. The phrasing of this tip I wrote doesn't make much sense outside of this context, which is how it'll be seen in game. Changing it to "Ives is a pet" is a bit more funny to me, and makes sense in the context of the round tip as seen in game. Reminding people Ives is in fact a pet is funny to me because its the one department pet I seriously haven't seen leave the RD's office more than three times in the 3 or so years I've been playing this server.